src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/Process_StreamTests.cs

```diff
@@ -9,105 +9,112 @@ namespace System.Diagnostics.ProcessTests
 {
     public partial class ProcessTest
     {
+
+        Process CreateProcessError()
+        {
+            Process p = new Process();
```

Comment:
Same comment as for CreateProcessInfinite.  These could all just be of the form:
```
Process p = CreateProcess();
p.StartInfo.Arguments = "error";
return p;
```
and if you changed CreateProcess to accept an optional string argument, these could all just be of the form:
```
return CreateProcess("error");
```